helpers/go/gofips.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+# helper script for Go FIPS support
+
+set -eu -o pipefail
+shopt -qs failglob
+
+GO_VERSION="${GO_VERSION:?}"
```

Comment:
Can we have a default value for this somehow? E.g. `/etc/go.default` that we read if `GO_VERSION` is not set? Then we can control the default from the Dockerfile and not have to update `macros/shared` every time.

Reply:
Sure, that could be setting `GO_VERSION` explicitly in the `sdk-golden` stage of the Dockerfile. Agreed that would be more appropriate than in `macros/shared`, as I've found not all go packages that we build with the SDK call the `_set_cross_go_env` macro.

This would also eliminate the need to make changes to Twoliter's `docker-go` cmd